Sources/AblyAssetTrackingSubscriber/DefaultSubscriber.swift

```diff
@@ -297,4 +298,9 @@ extension DefaultSubscriber: AblySubscriberServiceDelegate {
         logger.debug("subscriberService.didReceiveEnhancedLocation.", source: String(describing: Self.self))
         callback(event: DelegateEnhancedLocationReceivedEvent(location: location))
     }
+    
+    func subscriberService(sender: AblySubscriber, didReceiveResolution resolution: Resolution) {
+        logger.debug("subscriberService.didReceiveResolution.", source: String(describing: Self.self))
```

Comment:
Cosmetics, but why not "\(Self.self)"?

Reply:
no reason :) if it's a blocker I can change it if not I would leave it as it is